public/formEnumLookups.js

```diff
@@ -62,6 +62,17 @@ export const FILTER_ITEMS = {
   'nasopharyngeal swabs': {
     name: 'ftm-accept-item-nasopharyngeal-swabs',
     isSet: false
+  },
+  // France uses certain country specific naming for accepted items
+  'ffp2 masks (n95) or ffp3': {
+    name: 'ftm-accept-item-france-n95s',
+    isSet: false,
+    countrySpecific: 'fr'
```

Comment:
this is European countries, not just FR

Reply:
@drwestco any thoughts on the best way to handle this? At this point every country except for the US / CA would have this item on their form since in the EU N95 masks are called FFP2. Maybe a `countryBlacklist` array?

Reply:
Now that I think about it, as soon as we can migrate the US to use Merge Config this is a moot point since I believe it is the last holdout not using it. @awong-dev do you think this would be possible to do today?